packages/ipfs-core/src/components/version.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-const pkg = require('../../package.json')
+const pkg = require('../../../ipfs/package.json')
```

Comment:
This will fail at runtime as it traverses outside of the module folder.  E.g. if you only have `ipfs-core` installed (e.g. your app is running an in-process IPFS node and you don't need a CLI/HTTP API server/daemon, etc).

Reply:
A better solution may be to have the build write a version file out at build time with the version number, git hash, etc, then we can try to load it here.  It would also mean that the whole `package.json` doesn't get included in the browser bundle which would be a nice side benefit.